tests/www/api/experimental/test_dag_runs_endpoint.py

```diff
@@ -97,12 +97,8 @@ def test_get_dag_runs_success_with_state_no_result(self):
         # Create DagRun
         trigger_dag(dag_id=dag_id, run_id='test_get_dag_runs_success')
 
-        response = self.app.get(url_template.format(dag_id))
-        assert 200 == response.status_code
-        data = json.loads(response.data.decode('utf-8'))
-
-        assert isinstance(data, list)
-        assert len(data) == 0
+        with pytest.raises(ValueError):
```

Comment:
I have a slight doubt about this one. This is "technically" breaking change with the experimental API, but one that shold be very welcomed by any user. So I am tempted to leave it like that (I would treat it as a bugfix).

Reply:
What bit throws the value error here? (It's not clear to from just looking at the code)

Oh, that `state=dummy` results in a ValueError -- wouldn't that result in a 500 returned, when instead it should be a 400?